src/pd/async/client.rs

```diff
@@ -255,7 +80,9 @@ impl PdClient for RpcClient {
         req.set_store(stores);
         req.set_region(region);
 
-        let resp = try!(do_request(self, |client| client.Bootstrap(req.clone())));
+        let resp = try!(sync_request(&self.leader_client,
+                                     |client| client.Bootstrap(req.clone()),
+                                     LEADER_CHANGE_RETRY));
         try!(check_resp_header(resp.get_header()));
```

Comment:
Defer to @BusyJay  https://github.com/pingcap/tikv/pull/1712#discussion_r109278371